doc/functions/enum.rst

```diff
@@ -13,10 +13,10 @@
     {{ enum('App\\MyEnum').SomeCase.value }}
 
     {# get all cases of an enum #}
-    {% enum('App\\MyEnum').cases() %}
+    {% enum('App\\MyEnum').cases %}
 
     {# call any methods of the enum class #}
-    {% enum('App\\MyEnum').someMethod() %}
+    {% enum('App\\MyEnum').someMethod %}
```

Comment:
What about

```suggestion
    {% enum('App\\MyEnum').someMethod %}
    {% enum('App\\MyEnum').someMethod(someArg) %}
```

Reply:
That would work for me even if this is really just how method calls work by default in Twig, so nothing specific here.